diff --git a/src/rules/index.d.ts b/src/rules/index.d.ts
index d1e8e3fc4cbfd68c349e5dd688d0d67a6969dc39..8db4b5831847def4a9e6064ef8ad5f7632391e74 100644
--- a/src/rules/index.d.ts
+++ b/src/rules/index.d.ts
@@ -23,6 +23,9 @@ import type { VueI18nRules } from './vue-i18n';
 import type { VuePugRules } from './vue-pug';
 import type { YmlRules } from './yml';
 
+export * from './rule-config'
+export * from './rule-severity'
+
 /**
  * Rules.
  *
